demo/fiber/skynet_test/pom.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Does Loom provide the copyright note for this file?

Reply:
No, I didn't find any specific copyright note provided by Loom for these files. Should I just keep them empty?

Reply:
Sure.